app/views/substitutions/_form.html.haml

```diff
@@ -1,6 +1,5 @@
 = simple_form_for @substitution do |form|
   /= form.input :closed, disabled: false
-  = form.association :mentor 
```

Comment:
Plz don't remove it complete but only for the new action.

You can check if it is the new action using @substitution.new_record?